File: Jomel-Isis/jomel/lib/models/member.dart
class Member{
  final int id;
  final String name;
  final String password;
  final String nickname;
  final String picture;
  final String email;
  final bool isAdmin;

  Member({this.id, this.name, this.password, this.nickname, this.picture, this.email, this.isAdmin});

  factory Member.newMemberfromJson(Map<String, dynamic> json){
    return Member(
      id: json["Id"],
      name: json["Name"],
      password: json["Password"],
      email: json["Email"]
    );
  }
}

File: JomelAPI/JomelAPI/Models/Member.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace JomelAPI.Models
{
    public class Member
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string NickName { get; set; }
        public string Picture { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public bool isAdmin { get; set; }
        public ICollection<PointMembers> PointsTops { get; set; }
    }
}
